soc/ite/ec/it8xxx2/Kconfig

```diff
@@ -154,6 +154,13 @@ config SOC_IT8XXX2_JTAG_DEBUG_INTERFACE
 	  - GPIOA6 -> TRST
 	  Supported I/O voltage is 3.3V.
 
+config SOC_IT8XXX2_LCVCO
+	bool "LCVCO calibration"
```

Comment:
Should this have `depends on SOC_IT8XXX2_INT_32K`?  All the C build time checks require both symbols to be enabled.

Reply:
Updated. Thanks for the suggestion.